src/data/index.ts

```diff
@@ -363,16 +361,22 @@ export const exportCanvas = async (
 
 export const loadScene = async (
   id: string | null,
-  privateKey?: string | null,
-  initialData?: ImportedDataState,
+  privateKey: string | null,
+  // Supply initialData even if importing from backend to ensure we restore
+  //  localStorage user settings which we do not persist on server.
+  // Non-optional so we don't forget to pass it even if undefined
```

Comment:
```suggestion
  // localStorage user settings which we do not persist on server.
  // Non-optional so we don't forget to pass it even if undefined.
```

Reply:
This is something I do to visually group thoughts/items without using actual paragraphs (to save space). You didn't complain so far (I get 108 hits for `(//|\*) {2,}\S`) :).

Reply:
first time I'm noticing it.. it just looks weird indenting paragraphs..